### PR TITLE
Leadership documents

### PR DESCRIPTION
**Pull request by 1bigchonk**
Merging branch to chunk out better. Files that need review:

CFED.adoc
CFEC.adoc
Elections.adoc

Everything else should be branched into a new feature, but in smaller chunks.

### CODE REVIEW

File: Committees/National Committees/NPEC/NPEC.adoc [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+== Purpose
+
+Political education of PLTCom members is the sole mission of this committee. It will provide all members a course in capitalism first and scientific socialism second followed by a course on party work. It will also maintain a comprehensive digital library of marxist political material as well as a physical library as possible. 
+
+== Structure
+
+Only _full_ members can hold these positions:
+
+- Committee Chair
+- Librarian
```
Comment (mgcarbonell) [inline Committees/National Committees/NPEC/NPEC.adoc]: Where/what does the librarian do?

**Review by ghost — CHANGES_REQUESTED** [review]
Suggestions for the Chapter Firearms Education Director position document.

File: Committees/Chapter Committees/CFEC/CFEC.adoc [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+== Requirements
+// This section should describe the requirements that need to be met to join  this committee. For national or regional committees, include this section:
+// Joining this committee requires you fulfill the requirements listed here (link to relevant chapter committee director)
+To join the CFEC, you must complete the following PLTCOM courses:
+
+* PLTCOM Intro to Scientific Socialism
+* PLTCOM Intro to Firearms
+
+Course equivalencies may be accepted based on the judgement of the CFED and National Central Committee. No furthet experience with firearms is necessary.
```
Comment (ghost) [inline Committees/Chapter Committees/CFEC/CFEC.adoc]: furthet -> further

File: Committees/Chapter Committees/CFEC/CFEC.adoc [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+== Composition
+// Use this space to describe who the committee is composed of
+The CFEC is composed of chapter members who meet the requrements listed below in the requirements section, and have applied for the committee. The committee is lead by the <<CFED.adoc#,Chapter Firearms Education Director (CFED)>>
+
+== Requirements
+// This section should describe the requirements that need to be met to join  this committee. For national or regional committees, include this section:
+// Joining this committee requires you fulfill the requirements listed here (link to relevant chapter committee director)
+To join the CFEC, you must complete the following PLTCOM courses:
+
+* PLTCOM Intro to Scientific Socialism
```
Comment (ghost) [inline Committees/Chapter Committees/CFEC/CFEC.adoc]: We should standardize the course titles. Intro or Introduction?

Comment (ghost) [reply Committees/Chapter Committees/CFEC/CFEC.adoc]: Also, we probably don't need to double up on PLTCOM here

Comment (1bigchonk) [reply Committees/Chapter Committees/CFEC/CFEC.adoc]: Actually, I'd like course _numbers_ , but I think that's @Joedirte1917's department.

File: Committees/Chapter Committees/CFEC/CFEC.adoc [hunk 4]
```diff
@@ -0,0 +1,37 @@
+= Chapter Firearms Education Committee (CFEC)
+// Title of committee goes here
+:toc:
+
+== Composition
+// Use this space to describe who the committee is composed of
+The CFEC is composed of chapter members who meet the requrements listed below in the requirements section, and have applied for the committee. The committee is lead by the <<CFED.adoc#,Chapter Firearms Education Director (CFED)>>
```
Comment (ghost) [inline Committees/Chapter Committees/CFEC/CFEC.adoc]: requrements -> requirements

File: Committees/Chapter Committees/CFEC/CFEC.adoc (hunk 2 above)
Comment (ghost) [reply Committees/Chapter Committees/CFEC/CFEC.adoc]: judgement -> judgment

File: Committees/Chapter Committees/CFEC/CFEC.adoc [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+To join the CFEC, you must complete the following PLTCOM courses:
+
+* PLTCOM Intro to Scientific Socialism
+* PLTCOM Intro to Firearms
+
+Course equivalencies may be accepted based on the judgement of the CFED and National Central Committee. No furthet experience with firearms is necessary.  
+
+Further, members must submit an application to join the committee. The application asks the following questions:
+
+* How much experience do you have with firearms
```
Comment (ghost) [inline Committees/Chapter Committees/CFEC/CFEC.adoc]: Might want to add 'and what kind'

File: Committees/Chapter Committees/CFEC/CFEC.adoc [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+* How many hours per week, on average, are you willing to commit to this committee
+* How many committees are you already a member of
+
+These questions are meant to assess a members ability to participate in the committee and to develop a training plan for a members growth. Importantly, members will **NOT** be excluded for lack of firearms experience.
```
Comment (ghost) [inline Committees/Chapter Committees/CFEC/CFEC.adoc]: a members ability -> the applicant's ability
a members growth -> the further development of the applicant's firearms proficiency, with the aim of heightening their capacity to contribute.

File: Committees/Chapter Committees/CFEC/CFEC.adoc [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+== Meetings
+// Describe the typical meeting and the frequency
+The CFEC should meet monthly to discuss the progress on the goals set in the One Year Plan
```
Comment (ghost) [inline Committees/Chapter Committees/CFEC/CFEC.adoc]: at least on a monthly basis

File: Committees/Chapter Committees/CFEC/CFEC.adoc (hunk 4 above)
Comment (1bigchonk) [inline Committees/Chapter Committees/CFEC/CFEC.adoc]:
```suggestion
The CFEC is composed of chapter members who meet the requirements listed below in the requirements section, and have applied for the committee. The committee is lead by the <<CFED.adoc#,Chapter Firearms Education Director (CFED)>>
```

File: Committees/Chapter Committees/CFEC/CFEC.adoc (hunk 2 above)
Comment (1bigchonk) [inline Committees/Chapter Committees/CFEC/CFEC.adoc]:
```suggestion
Course equivalencies may be accepted based on the judgment of the CFED and National Central Committee. No further experience with firearms is necessary.  
```

File: Committees/Chapter Committees/CFEC/CFEC.adoc (hunk 5 above)
Comment (1bigchonk) [inline Committees/Chapter Committees/CFEC/CFEC.adoc]:
```suggestion
* How much, and what kind of, experience do you have with firearms?
```

File: Committees/Chapter Committees/CFEC/CFEC.adoc (hunk 6 above)
Comment (1bigchonk) [inline Committees/Chapter Committees/CFEC/CFEC.adoc]:
```suggestion
These questions are meant to assess the applicant's ability to participate in the committee and to develop a training plan for the further development of the applicant's firearms proficiency, with the aim of heightening their capacity to contribute. Importantly, members will **NOT** be excluded for lack of firearms experience.
```

File: Committees/Chapter Committees/CFEC/CFEC.adoc (hunk 7 above)
Comment (1bigchonk) [inline Committees/Chapter Committees/CFEC/CFEC.adoc]:
```suggestion
The CFEC should meet at least monthly to discuss the progress on the goals set in the One Year Plan
```

File: Committees/Chapter Committees/CFEC/CFEC.adoc [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+== Duties
+// Describe the duties of members on this committee
+
+* Assist the CFED in the drafting of the committee's One Year Plan
+* Coordinate with the CFED on a personal training plan, then execute the goals on the personal training plan
```
Comment (ghost) [inline Committees/Chapter Committees/CFEC/CFEC.adoc]:
```suggestion
* After coordinating with the CFED on a personal training plan, successful completion of the goals enumerated therein
```

File: Committees/Chapter Committees/CFEC/CFED.adoc [hunk 9]
```diff
@@ -0,0 +1,25 @@
+// Title of leadership position goes here
+= Chapter Firearms Education Director (CFED)
+:toc:
+
+== Requirements
+// List any requirements for leading the committee or working group
+* Sufficient training to conduct at least the Intro to Firearms course, with a plan for more advanced training
+* Completion of PLTCOM Introduction to Scientific Socialism
+
```
Comment (1bigchonk) [inline Committees/Chapter Committees/CFEC/CFED.adoc]:
```suggestion
* Membership in a PLTCOM-recognized Marxist-Leninist party or pre-party formation;footnote:[If there is at least one nominee meeting this recommendation, it becomes a requirement.] 
* Additional training permitting the instruction of more advanced courses.
```

File: Committees/Chapter Committees/CFEC/CFEC.adoc (hunk 2 above)
Comment (1bigchonk) [reply Committees/Chapter Committees/CFEC/CFEC.adoc]:
```suggestion
Course equivalencies may be accepted based on the judgment of the CFED and National Central Committee. No further experience with firearms is necessary.  
```

File: Committees/Chapter Committees/CFEC/CFEC.adoc (hunk 4 above)
Comment (1bigchonk) [reply Committees/Chapter Committees/CFEC/CFEC.adoc]:
```suggestion
The CFEC is composed of chapter members who meet the requirements listed below in the requirements section, and have applied for the committee. The committee is lead by the <<CFED.adoc#,Chapter Firearms Education Director (CFED)>>
```

File: Committees/Chapter Committees/CFEC/CFED.adoc [hunk 10]
```diff
@@ -0,0 +1,26 @@
+// Title of leadership position goes here
+= Chapter Firearms Education Director (CFED)
+:toc:
+
+== Requirements
+// List any requirements for leading the committee or working group
+* Sufficient training to conduct at least the Intro to Firearms course, with a plan for more advanced training
+* Completion of PLTCOM Introduction to Scientific Socialism
+* Membership in a PLTCOM-recognized Marxist-Leninist party or pre-party formation;footnote:[If there is at least one nominee meeting this recommendation, it becomes a requirement.]
```
Comment (1bigchonk) [inline Committees/Chapter Committees/CFEC/CFED.adoc]:
```suggestion
* Membership in a PLTCOM-recognized Marxist-Leninist party or pre-party formation footnote:[If there is at least one nominee meeting this recommendation, it becomes a requirement.] 
```

File: Committees/Chapter Committees/CFEC/CFED.adoc [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+= Chapter Firearms Education Director (CFED)
+:toc:
+
+== Requirements
+// List any requirements for leading the committee or working group
+* Sufficient training to conduct at least the Intro to Firearms course, with a plan for more advanced training
+* Completion of PLTCOM Introduction to Scientific Socialism
+* Membership in a PLTCOM-recognized Marxist-Leninist party or pre-party formation footnote:[If there is at least one nominee meeting this recommendation, it becomes a requirement.] 
+* Additional training permitting the instruction of more advanced courses.
+== Duties
```
Comment (1bigchonk) [inline Committees/Chapter Committees/CFEC/CFED.adoc]:
```suggestion

== Duties
```

File: Committees/Committee-Process.adoc [hunk 12]
```diff
@@ -0,0 +1,84 @@
+= Committee Process
+
+This document outlines the rough process committees should implement to effectively set and meet goals.
```
Comment (ghost) [inline Committees/Committee-Process.adoc]:
```suggestion
This document provides a general outline of the process committees should implement to effectively set and meet goals.
```

File: Committees/Committee-Process.adoc [hunk 13]
```diff
@@ -0,0 +1,84 @@
...
+
+== Conduct an Investigation
+
+After being elected, new committee directors should investigate the needs of the organization. This can be done by polling the members of your committee, who should have a better understanding of what's needed at the ground level, or by polling the membership directly using the forms service built in to Chapterbox*. You should ask open ended questions like:
```
Comment (ghost) [inline Committees/Committee-Process.adoc]:
```suggestion
After being elected, new committee directors should investigate the needs of the organization. Recommended approaches include polling the members of your committee, who should have a better understanding of what is needed at the ground level, and by polling the membership directly using the forms service incorporated within Chapterbox*. You should ask open-ended questions like:
```

**Comment by 1bigchonk** [comment]
@dialecticfire @Joedirte1917 modified the elections document to include the application process for chapter committee directors (wording can be changed for release to describe our current situation). 

Also modified the course descriptions in the CFED document

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Review by mgcarbonell — APPROVED** [review]

**Review by logicpriest — APPROVED** [review]

**Pull request merged**